src/printing.py

```diff
@@ -12,6 +12,11 @@
 # y, and z:
 # x is 10, y is 2.25, z is "I like turtles!"
 
+print("x is %d, y is %f, z is %s" % (x, round(y, 2), z))
+
 # Use the 'format' string method to print the same thing
+print("x is {}, y is {}, z is {}".format(x, round(y, 2), z))
```

Comment:
This works, but there's also a way to do the rounding inside the formatted string itself. Check out this link for the first two problems https://mkaz.blog/code/python-string-format-cookbook/

Reply:
Awesome, I'll change it. That syntax is much nicer.